### ISSUE TITLE
Feature:  testing suite

### ISSUE THREAD

**Issue by andypohl** [pegasus-isi/dipa-workflow#6]
I'm just putting this here as a reminder that once we're at a 1.0, i.e. the whole workflow is running, we should probably build up a test suite with toy (small) data, so if we make future changes, we'll see if something unexpected breaks.  A test suite could be very helpful in this stage too, but the main priority now is still building up the workflow piece by piece.  Test suites can be produced to:
1. Check individual scripts.
2. Make sure changes to 3rd-party BI software don't affect DIPA/Perseus.
3. Make sure Pegasus changes don't affect anything.  

I'll try to find some good examples and in particular try to put the pegasus part in place.

**Comment by AndrewJSchoen** [comment]
Definitely. This is an absolute must. The tricky thing will be getting some datasets that don't have participant data in them. Perhaps some non-human phantom data would work well.
